sdk/core/core-http/lib/policies/exponentialRetryPolicy.ts

```diff
@@ -50,6 +50,42 @@ const DEFAULT_CLIENT_RETRY_COUNT = 3;
 const DEFAULT_CLIENT_MAX_RETRY_INTERVAL = 1000 * 90;
 const DEFAULT_CLIENT_MIN_RETRY_INTERVAL = 1000 * 3;
 
+/**
+ * Options that control how to retry failed requests.
+ */
+export interface RetryOptions {
+  /**
+   * The maximum number of retry attempts.  Defaults to 3.
+   */
+  retryCount?: number;
+
+  /**
+   * The amount of delay in milliseconds between retry attempts. Defaults to 30000
+   * (30 seconds). The delay increases exponentially with each retry up to a maximum
+   * specified by maxRetryDelayInMs.
+   */
+  retryIntervalInMs?: number;
+
+  /**
+   * The minimum delay in milliseconds allowed before retrying an operation. Defaults
+   * to 3000 (3 seconds).
+   */
+  minRetryDelayInMs?: number;
```

Comment:
The current way the delays are calculated, the delay before the first retry happens to be `minRetryDelayInMs + retryIntervalInMs * (number between 0.8 and 1.2))`. So the docs here that say that `minRetryDelayInMs` is the minimum delay itself is a wrong statement 

I want to propose removing the `minRetryDelayInMs` property and renaming the `retryIntervalInMs` to `retryDelayInMs`. We want to start the delay between retries from the value provided in `retryDelayInMs` and increment it exponentially between subsequent retries.

The `retryDelayInMs` in itself should act as the min value and we shouldn't need another property to configure the min.

Reply:
This proposal sounds fine to me, I'll make the changes if @bterlson concurs

Reply:
I like @ramya-rao-a's approach and it aligns with what we did in EventHubs too.